docs/setup/requirements.md

```diff
@@ -98,8 +98,9 @@ To build and test libcu++ yourself, you will need the following in addition to
   - We recommend installing lit using Python's pip package manager.
 
 
-[API section]: ./api.md
-[synchronization library section]: https://nvidia.github.io/libcudacxx/api/synchronization_library.html
+[Standard API section]: ./standard_api.md
+[Extended API section]: ./extended_api.md
+[synchronization library section]: https://nvidia.github.io/libcudacxx/standard_api/synchronization_library.html
```

Comment:
This should be a relative link.

Reply:
No, these all need to be non-relative links, because the README.md is a symlink; they would be incorrect when rendered on GitHub otherwise.

Reply:
Oh this isn't in the readme. Probably a copy/paste issue then.